spec/requests/red_hat_migration_analytics_manifest_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe "Red Hat  Migration Analytics Manifest API" do
+  let(:api_red_hat_migration_analytics_url) { "#{api_url}/migration_analytics" }
+  let(:manifest) { { "ManageIQ::Providers::Vmware::InfraManager" => {}} }
+  let(:task) { FactoryBot.create(:miq_task) }
+
+  describe "POST" do
+    it "/api/red_hat_migration_analytics/:id bundle action" do
+      api_basic_authorize "red_hat_migration_analytics"
+
+      ems1 = FactoryBot.create(:ems_vmware, :name => "sample vmware1")
+
+      allow(Cfme::CloudServices::InventorySync).to receive("bundle_queue").with(@user.userid, manifest, [ems1.id]) { task.id }
+
+      post(api_red_hat_migration_analytics_url(nil, manifest, ems1),
```

Comment:
ok, got confused here (because of line 2 above), just change to post(api_red_hat_migration_analytics_url,
we should just be hitting /api/red_hat_migration_analytics.